adapter-pinecone/.gitignore

```
@@ -0,0 +1,38 @@
+target/
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**/target/
+!**/src/test/**/target/
+
+### IntelliJ IDEA ###
+.idea/modules.xml
+.idea/jarRepositories.xml
+.idea/compiler.xml
+.idea/libraries/
+*.iws
+*.iml
+*.ipr
+
+### Eclipse ###
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
+
+### NetBeans ###
+/nbproject/private/
+/nbbuild/
+/dist/
+/nbdist/
+/.nb-gradle/
+build/
+!**/src/main/**/build/
+!**/src/test/**/build/
+
+### VS Code ###
+.vscode/
+
+### Mac OS ###
+.DS_Store
```

If you prefix it with a `*`, it would happily take care of it on all directories

```suggestion
*.DS_Store
```

---

Why not update the core .gitignore at the root of the project instead?

---

I'm not clear if there is a change being requested here or not, please update to let me know?

---

Yes, change requested. There is a .gitignore at the root of the project, and it should be adjusted rather than propogage more .gitignore files to manage.